pkg/cstorvolumereplica/v1alpha1/cstorvolumereplica.go

```diff
@@ -25,12 +28,24 @@ type cvrList struct {
 func (l *cvrList) GetPoolUIDs() []string {
 	var uids []string
 	for _, cvr := range l.items {
-		uid := cvr.object.GetLabels()[cstorPoolUIDLabelKey]
+		uid := cvr.object.GetLabels()[string(cstorPoolUIDLabel)]
 		uids = append(uids, uid)
 	}
 	return uids
 }
 
+// GetPoolName returns a list of cstor pool
+// name corresponding to cstor volume replica
+// instances
+func (l *cvrList) GetPoolNames() []string {
```

Comment:
Create another function called `ListUniquePoolNames` and make use of it in test code

Reply:
done